docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,162 @@ described below, and relative to the joint TMREL.
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+
+Vivarium modelling strategy
++++++++++++++++++++++++++++
+
+First, we convert the GBD 500g-2weeks birthweight-ga bins/categories to a joint continuous distribution using Abie's notebook (add link?). We assume a uniform distribution within each bin/category. 
+
+.. note ::
+    That this is likely biasing towards overestimating extreme birthweights or gestational ages. For example, in the 0-500g category, most babies are probably pretty close to 500g, not equally probable to be 1 gram versus 499 grams.
+
+Because the relative risks from GBD are for all-cause mortality in the early and late neonatal period, we first define all-cause mortality rate (ACMR) as the sum of: 
+
+   - mortality from causes that are affected by lbwsg and modelled in the sim (green)
+   - mortality from causes that are affected by lbwsg but not modelled in the sim (blue)
+   - mortality from causes that are unaffected by lbwsg, modelled or not modelled (salmon)
+
+An example of these causes from the large-scale-food fortification concept model diagram is shown below: 
+
+.. image:: causes_equation.svg
+
+All-cause mortality is sum of all cause-specific mortalities: 
+
+   ACMR =  :math:`\sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{CSMR}`
+
+The mortality from unmodelled causes affected by lbwsg (blue) is thus: 
+
+   ACMR - :math:`\sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR}`
+
+We are interested in applying the PAF and relative risk to only the causes that GBD considers to be affected by lbwsg (green and blue):
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in early and late neonatal period for category i
+   |  rr_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+   |  PAF* = this is the PAF of the most detailed cause affected by lbwsg
+
+Hence, the mortality hazard for an individual in category i is:  
+
+:math:`\sum\limits_{\text{salmon}}\text{CSMR} + (ACMR - \sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+
+.. important :: 
+
+   PAF in the above equation represents PAFs for most-detailed-cause (they are all roughly equal) affected by LBWSG (or as calculated in vivarium from LBWSG relative risks and exposure). This approach assumes that relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG (green and blue causes). 
```

Comment:
Why is this important?  What are the alternatives (e.g. what PAF would we use besides the one for most-detailed causes)?

Reply:
all-cause PAF, which is a different value. WE have to use the most-detailed PAF because this is the one that is calculated directly from the relative risks, which is what we want to apply to the entire group of LBWSG affected causes

Reply:
ill include link to the notebook that calculates the all-cause PAF and most-detailed-cause PAF

Reply:
how do I reference the notebook? 

:ref:lbwsg_paf_notebook <_where do I find how this is called?_>

Reply:
> all-cause PAF, which is a different value. WE have to use the most-detailed PAF because this is the one that is calculated directly from the relative risks, which is what we want to apply to the entire group of LBWSG affected causes

I think some version of this explanation should appear within the `.. important::` directive, as well as linking to the notebook.  I'm not sure where the notebook is - @alibow is it somewhere in the vivarium_data_analysis repo?

Reply:
Yes it is!! it's in the data analysis repo

Reply:
Great, then to reference it, you would just do :ref:`LBWSG PAF notebook <https://github.com/ihmeuw/vivarium_data_analysis/blob/master/pre_processing/lbwsg/LBWSG%20exposure%2C%20rrs%2C%20pafs.ipynb>` (or replace the text before <...> with whatever you want).

Reply:
the :ref: doesnt work to link a website. Instead, I used: 

`lbwsg PAF notebook <https://github.com/ihmeuw/vivarium_data_analysis/blob/master/pre_processing/lbwsg/LBWSG%20exposure%2C%20rrs%2C%20pafs.ipynb>`__.

https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html#cheat-sheet-intersphinx

Reply:
Oh, right -- sorry, I got confused.  The :ref: syntax is specific to Sphinx, but the backtick ... backtic_ syntax is pure .rst